Mathlib/MeasureTheory/Integral/Lebesgue/Basic.lean

```diff
@@ -277,6 +277,10 @@ lemma setLIntegral_eq_zero {f : α → ℝ≥0∞} {s : Set α} (hs : Measurable
 
 section
 
+theorem lintegral_eq_zero_of_ae_eq_zero {f : α → ℝ≥0∞} :
+    f =ᵐ[μ] 0 →  ∫⁻ a, f a ∂μ = 0 :=
```

Comment:
How about making the left hand side a hypothesis, per "hypothesis left of colon"?